porting/build-sources.rst

```diff
@@ -63,8 +63,17 @@ To check which config options needs to be adjusted we use `mer-kernel-check <htt
 
 If you don't know the path to your kernel config run ``grep "TARGET_KERNEL_CONFIG" device/<VENDOR>/<CODENAME>/BoardConfig.mk``. It should be in ``arch/arm/configs/<CONFIG>`` or ``arch/arm64/configs/<CONFIG>`` depending on the architecture of your device.
 
+.. warning::
+		Make sure that your configuration changes are not overridden by later lines in the config file.
+		
+		Also be aware that ``# CONFIG_IKCONFIG_PROC is not set`` *may* look like a harmless comment, `but it actually unsets`__ ``CONFIG_IKCONFIG_PROC`` if it is set (e.g. by an edit you made earlier in the file).
```

Comment:
Are we positive about this? I do have the *feeling* that # CONFIG_BLA is not set does *something* but I haven't seen any reference for this. The reference given here, actually states that the presence of such a line and the absence of such a line are equivalent. Does anyone have a reference clarifying that is not set actually does something?